string header
float32 x
float32 y
float32 width
float32 height
float32 angle
string desc_1
string desc_2

